### DISCUSSION TITLE
dashboard external-logs not working

### DISCUSSION THREAD

**Discussion by cizara** [tektoncd/community#836] [Q&A]
Hi, I'm trying to access backed up logs from the Tekton dashboard by adding the arg `--external-logs=http://logs-server:3000` to the dashboard deployment, but can't make it work and every old log shows `No log available`.

Some more info:
- if I create an ingress for the service at `logs-server:3000` and try to access it I can see the logs
- it also shows the logs if I access the dashboard endpoint  `/v1/logs-proxy/tekton-pipelines/7f343d52-3e5f-11ed-88e2-564b03a243f1-docker-build-pod/step-docker-build`
- looking at the requests in the `logs-server` service, there isn't any call from the Tekton dashboard
- Tekton dashboard logs don't show any error

Any ideas on what can be the problem here?

**Comment by cizara** [comment]
one thing that I notice is that in the Network tab of the browser's dev tools, there are several web socket calls with `101` HTTP response. Most of them have this response message:
```
{
  "type": "ERROR",
  "object": {
    "kind": "Status",
    "apiVersion": "v1",
    "metadata": {},
    "status": "Failure",
    "message": "too old resource version: 14069611 (14957714)",
    "reason": "Expired",
    "code": 410
  }
}
```
not sure if this is realted to the my problem or not

**Reply by AlanGreene** [reply]
Responded on Slack: https://tektoncd.slack.com/archives/CHTHGRQBC/p1664459121958499
